XO.py

```diff
@@ -0,0 +1,157 @@
+from random import shuffle
+
+# size of board
+N = 3

```

Comment:
Very cool you made the board size generic!

Reply:
Thanks :)